ci-operator/config/openshift/cluster-network-operator/openshift-cluster-network-operator-master.yaml

```diff
@@ -339,20 +283,6 @@ tests:
     - chain: baremetalds-ipi-test
     workflow: baremetalds-e2e-ovn-ipv6
   timeout: 5h0m0s
-- as: e2e-network-mtu-migration-sdn-ipv4
```

Comment:
This is redundant with the ovn migration anyway isn't it? (Since sdn-to-ovn requires decreasing the MTU for Geneve.)

Reply:
I don't know the answer for sure, but I think it is. Anyway, redundant or not, it seems fine to kill.